src/presentation/http/router/note.test.ts

```diff
@@ -244,7 +243,128 @@ describe('Note API', () => {
         expect(response?.json().accessRights).toStrictEqual({ canEdit: true });
       });
     });
+  });
+
+  describe('PATCH note/:notePublicId ', () => {
```

Comment:
please add test case checking the behaviour of the endpoint when trying to update not existing note